nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
Sorry, I missed the part where t_slice_begin_ and t_slice_end_ are changed. t_slice_begin_ is also updated in line 593 (approx. in master). The initialization with timeval() is also fine according to C++ standard. Thus no reason to worry.